src/AmsterdamPHP/Bundle/MeetupBundle/Command/CreateJoindinPlaceholderCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace AmsterdamPHP\Bundle\MeetupBundle\Command;
+
+use CL\Slack\Payload\ChatPostMessagePayload;
+use Codeliner\ArrayReader\ArrayReader;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CreateJoindinPlaceholderCommand
+ */
+class CreateJoindinPlaceholderCommand extends ContainerAwareCommand
+{
+    /**
+     * @var \AmsterdamPHP\Bundle\MeetupBundle\Service\EventService
+     */
+    protected $meetup;
+
+    /**
+     * @var \CL\Slack\Transport\ApiClientInterface
+     */
+    protected $slack;
+
+    /**
+     * @var \GuzzleHttp\Command\Guzzle\GuzzleClient
+     */
+    protected $joindin;
+
+    /**
+     * @see Command
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('amsphp:joindin:create-placeholder')
+            ->setDescription('Creates the monthly meeting event at joind.in');
+    }
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        parent::initialize($input, $output);
+
+        $this->meetup = $this->getContainer()->get('meetup.events');
+        $this->slack = $this->getContainer()->get('cl_slack.api_client');
+        $this->joindin = $this->getContainer()->get('joindin.api')->getService(new \Joindin\Api\Description\Events());
+    }
+
+    /**
+     * @see Command
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
ideally i encapsulate all logic in my domain, given that i don't have nearly as much time to invest on this site, i operate on pragmatic, so we would extract logic if its actually used somewhere else.